packages/diagram/src/module/diagrams/content/common/defaultStyles.ts

```diff
@@ -0,0 +1,50 @@
+import { InterpreterModule, parse } from "@hylimo/core";
+
+/**
+ * Module providing default styles
+ */
+export const defaultStylesModule = InterpreterModule.create(
+    "common/defaultStyles",
+    [],
+    [],
+    [
+        ...parse(
+            `
+                scope.styles {
+                    vars {
+                        primary = config.primaryColor
+                        background = config.backgroundColor
+                        strokeWidth = 2
+                        subcanvasMargin = 40
+                    }
+                    type("span") {
+                        fill = var("primary")
+                        fontSize = 16
```

Comment:
well it already is configurable
you would just need to add a
```
styles {
    type("span") {
        fontSize = 20
    }
}
```
to your diagram
or do you mean via a variable? if you want to, I could do that (however as nothing overwrite this currently it would not really make a difference)

Reply:
> do you mean via a variable

Yes. Some people would like to have larger text by default.

Reply:
if you prefer a variable, here you go: 1dbd203d8081c0a256cbc232443c4150ce611fdf